decentraland/unity-renderer
# style: external URL prompt

### What does this PR change?
In the current version the pop up has too much irrelevant information when what we want to make explicit is that the scene is trying to take the user to another domain.
Comparing to the previous design the UI assets have been updated for the newest in the guidelines design. The irrelevant texts and the image were removed. The distances between text boxes and their sizes were adjusted in order to make all the UI look visually harmonious, clean and clear.

Before vs. after
<img width="746" alt="Screen Shot 2022-06-10 at 13 43 04" src="https://user-images.githubusercontent.com/51088292/173136961-1cd1b26f-53d3-424e-adf9-a1283fd9f8da.png">

### How to test?
1- Go to: https://play.decentraland.zone/?renderer-branch=style/LeaveExplorerPopup
2- Click on the pride calendar icon near the mini map (heart). 
3- Click an info button of any of the events to see the new pop up is working properly and looking good.
